portfolio/src/main/webapp/index.html

```diff
@@ -1,147 +1,154 @@
 <!DOCTYPE html>
 <html>
-  <head>
```

Comment:
Hi did you change the indentation? As `<head>` is a child of `<html>` it should be indented with 2 spaces. Changing the indentation is not needed in this case and actually makes the review a little bit harder since it highlights a lot of lines instead of only the ones where we need to focus.

Reply:
Yes, maybe I did reformatting code in intellij accidentally ...